### DISCUSSION TITLE
Where do you get the icons?

### DISCUSSION THREAD

**Discussion by daniel4-scratch** [ScratchAddons/ScratchAddons#1081] [Help] [answered]
Just a quick question where do you get the icons from, it would be great to know

**Comment by ghost** [comment]
What icons?

**Comment by daniel4-scratch** [comment]
Like the message icons
Those icons next to the tools
![8DFB070A-D5E5-4081-A01F-851C7BD2745D](https://user-images.githubusercontent.com/65277548/102826033-0652a600-4434-11eb-8ba3-449618c49bd8.jpeg)

**Reply by ghost** [reply]
Right click the icon and click Save image as.

**Reply by daniel4-scratch** [reply]
No, like where did you fetch or get the icons from originally?

**Reply by ghost** [reply]
Idk. @WorldLanguages, where do the icons come from?

**Comment by GrahamSH-LLK** [comment, marked as answer]
We use the UIL iconset (unicons) from Iconify.

**Reply by daniel4-scratch** [reply]
Thanks :)
